src/cpp/include/openvino/genai/continuous_batching_pipeline.hpp

```diff
@@ -56,6 +56,10 @@ class OPENVINO_GENAI_EXPORTS ContinuousBatchingPipeline {
 
     PipelineMetrics get_metrics() const;
 
+    std::vector<std::string> get_model_configuration();
+
+    void print_model_configuration();
```

Comment:
I think that instead of adding debug API to public API, we can either:
- Enable compiled time cmake option to print some debug information. 
- Or do it via environment variable. In this case we need to ensure that debug info handling does not affect performance.

It can be useful for timers which are currently profile each iteration / step. I vote for cmake option with debug information.

Reply:
I have moved the functionality to utils as we want to have it every time we use cb_pipeline.